apps/src/netsim/dataConverters.js

```diff
@@ -104,12 +110,11 @@ exports.alignDecimal = function (decimalString) {
     return prev;
   }, 0);
 
-  var nbspChar = "\xA0";
-  var nbspPadding = new Array(mostDigits + 1).join(nbspChar);
+  var zeroPadding = new Array(mostDigits + 1).join('0');
 
   return numbers.map(function (numString) {
     // Left-pad each number with non-breaking spaces up to max width.
```

Comment:
Is a polyfill okay?  ECMAScript 6 has [String.prototype.repeat (MDN)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat).